### ISSUE TITLE
Port PeerBlock to Mac / Apple OSX

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#161]
```
Would like to see a Macintosh Port of PeerBlock.

Currently running 10.6 Snow Leopard.

Thanks!
```

Original issue reported on code.google.com by `brando...@gmail.com` on 12 Oct 2009 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
i would also love a mac edition.  i am running peerguardian and it is just not 
what
it used to be.  im sure if others that are using peerguardian hear about 
peerblock
they will be interested too!
```

Original comment by `alex.she...@gmail.com` on 19 Oct 2009 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
Issue 169 has been merged into this issue.
```

Original comment by `peerbloc...@gmail.com` on 22 Oct 2009 at 5:24

**Comment by GoogleCodeExporter** [comment]
```
Updating the Summary line to make it easier for people to find this bug.

This is unfortunately going to be a long time coming at this point.  The code 
right
now has lots of Windows-specific stuff sprinkled throughout; this code will 
need to
be changed before we can even think about creating a Mac version of PeerBlock.  
We
will be doing this naturally as we start moving towards PeerBlock 2.0, at which 
point
we will be in a much better position to start considering a Mac port.

None of the developers currently own a Mac, either, so once the code is ready 
for
work to begin we'll need either to find a Mac developer who'd be interested in 
the
project, or else hope that PeerBlock is well-funded enough by then that we can 
afford
to use some money to purchase a development machine.

If you're interested you should definitely "Star" this issue though, as the more
Stars we see the more interested we'll assume our user community would be in 
such a
thing!  And if you're a Mac developer who'd be willing to help out - even just 
in an
advisory role, if you're not willing to commit to sinking much time into the 
project
- please let me know!  (http://tinymailto.com/peerblock)
```

Original comment by `peerbloc...@gmail.com` on 22 Oct 2009 at 5:30
- Changed title: **Port PeerBlock to Mac / Apple OSX**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Mac version please.
```

Original comment by `timdeso...@gmail.com` on 7 Nov 2009 at 6:56

**Comment by GoogleCodeExporter** [comment]
```
peerguardian already has a version for mac, can't someone hack it to use updated
lists meant for peerblock?
```

Original comment by `ebooma...@gmail.com` on 11 Nov 2009 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
It's not actively developed though, that's what is so appealing about 
Peerblock. Looking forward to a Mac 
version!
```

Original comment by `brando...@gmail.com` on 11 Nov 2009 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Please develop a Mac version. Running OSX 10.5 soon to be OSX10.6

Thanks
```

Original comment by `michaelc...@brktel.on.ca` on 15 Nov 2009 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
Peerblock MacOS X 10.6.2+ please! :):)
```

Original comment by `bestma...@gmail.com` on 15 Nov 2009 at 8:49

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I hope there is an os x 10.5/10.6 version developed soon.
please and thank you.
```

Original comment by `suzukasp...@gmail.com` on 19 Nov 2009 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
mac os x 10.6.2 please
```

Original comment by `trebor...@gmail.com` on 21 Nov 2009 at 3:32

**Comment by GoogleCodeExporter** [comment]
```
Yeah Pretty Please????
```

Original comment by `ScottyC...@gmail.com` on 22 Nov 2009 at 5:42

**Comment by GoogleCodeExporter** [comment]
```
PLEASE port to mac!!! Definitely need this!
```

Original comment by `PaulAndr...@gmail.com` on 3 Dec 2009 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
For those who want a Mac version, try the one which already exists at 
http://forums.phoenixlabs.org/forumdisplay.php?f=65 in the sticky

It may not be updated as regularly as PeerBlock but it should work.
```

Original comment by `nightstalkerz` on 3 Dec 2009 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
please help out the mac-heads

```

Original comment by `GrahamGu...@gmail.com` on 8 Dec 2009 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
To everyone who makes comments in this topic: "If you're interested you should 
definitely STAR this issue 
though, as the more STARS we see the more interested we'll assume our user 
community would be in such a
thing!". So please remember to star this topic!! ***
```

Original comment by `rosted` on 8 Dec 2009 at 8:57

**Comment by GoogleCodeExporter** [comment]
```
97 stars! WooT! Hopefully we can get some attention now and get a Mac Port! 
Definitely looking forward to this 
app! Thanks guys!
```

Original comment by `brando...@gmail.com` on 8 Dec 2009 at 5:15

**Comment by GoogleCodeExporter** [comment]
```
Believe me, we definitely understand that there are a *large* number of users
interested in a Mac version of PeerBlock!  

The thing to remember, though, is that as per Comment #3 it will unfortunately 
be
awhile before the code's ready to begin a port.  At this point we're probably 
1-2
months away from a PeerBlock 1.1 release, at which point we'll start working on
PeerBlock 1.2 (estimated 3-4 months of effort) as well as PeerBlock 2.0 in the
background . . . with PeerBlock 2.0 being the only thing we're working on after 
1.2
is released.  As we work on PeerBlock 2.0 we'll be getting the code into a shape
where it should be possible to begin a port to Mac.  I estimate a stable 
PeerBlock
2.0 coming out probably 6-9 months after 1.2, though Beta Releases should be
available before then.

As we start getting closer to starting in on this Mac port we will still need 
to have
some machine(s) upon which we'll be able to develop.  One user has offered VNC 
use of
his machine for development purposes, which should at least give us one 
platform upon
which we can work.  One other user had initially offered to donate his older 
Mac (a
G4 dual-processor machine, as I recall) so long as we paid shipping, but I never
heard back from him after his initial email.  Worst-case scenario we'll 
hopefully be
in a position where we'll be able to afford purchasing one or more
low-end/refurbished/used Macs at some point.

If we get more developers on board hopefully the above "tentative schedule" 
will be
able to be brought in a bit, but seriously . . . there's a *lot* of work 
that'll be
required to rearchitect the code to be able to support non-Windows OSes.

That said, thank you all for expressing your interest in this Mac port!  And
definitely continue Starring the issue - the more requests we see for this 
feature
the more "hot to trot" we'll be to get to it, and additionally we'll post here 
once
we start in on that project to hopefully get some of you guys to come on board 
as
Alpha-stage testers.
```

Original comment by `peerbloc...@gmail.com` on 8 Dec 2009 at 5:52
- Changed state: **Accepted**
- Added labels: **Milestone-2.0**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Thank you for outlining the Mac OS X version/port of PeerBlock.
(I used to do mainframe/server SW development many years back, so I understand 
the
effort and time required ;)

I was happy to find PeerGuardian some while back, for my ancient Windows XP 
minitower.
Now that I have managed to replace my equally ancient --and decommissioned :( --
iMac, with a Mac Mini, I look forward to the Mac OS X version of PeerBlock.

Thank you for your work.

(repost, as I noticed that adding a profile to my google ID allows my full, real
email address to be displayed! What a F disaster...
How do people here manage to make comments with NO email to be displayed?
Sorry to ask these dumb question?
```

Original comment by `beyoun...@gmail.com` on 10 Dec 2009 at 2:18

**Comment by GoogleCodeExporter** [comment]
```
I need a Mac version, please!
```

Original comment by `Ben.jami...@gmail.com` on 16 Dec 2009 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
Add me to the list to see a mac version of peerblock.
```

Original comment by `darthfa...@gmail.com` on 28 Dec 2009 at 8:16

**Comment by GoogleCodeExporter** [comment]
```
wanted to say, i use Leopard (10.5.8) and I don't plan on switching to Snow 
Leopard.
so, developers, please consider porting to Mac in a way that will work on 
anything
starting with vers. 10.4 or so. that way it will be usable for more of us.

thanks! i tried using PG but it didn't seem to block anything.
```

Original comment by `beyond1l...@gmail.com` on 28 Dec 2009 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
Would happily donate to help with a Mac version.  Send us Mac people a request 
for
donations to a Mac machine for development! :)
```

Original comment by `snowtosurf` on 1 Jan 2010 at 9:57

**Comment by GoogleCodeExporter** [comment]
```
Please develop a mac version. Thanks!
```

Original comment by `ilampri...@gmail.com` on 4 Jan 2010 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
I also would be happy to make a donation to see a up-to-date mac version. 
Please!!! 
Thanks!!!
```

Original comment by `bikosunflower@gmail.com` on 4 Jan 2010 at 4:05

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the effort already put forth on a great piece of software, and 
please add me to the chorus of voices 
requesting a Mac client.

Thank You!
```

Original comment by `lzt...@gmail.com` on 5 Jan 2010 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
Please add one more to the "Port PeerBlock to Mac" list.  :)  I will gladly 
donate $5 or so to get this on my Mac.
```

Original comment by `jameswes...@gmail.com` on 9 Jan 2010 at 7:04

**Comment by GoogleCodeExporter** [comment]
```
Another for mac port, $5, I would probably give $10 on non crashing beta 
delivery :D

and im bankrupt.  lol.

so yeah we want it.  Add 10 for me and the people like me who rarely say sh*t.

Thanku
```

Original comment by `ipc...@gmail.com` on 12 Jan 2010 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
it would be awesome to have peerblock for mac! thanks!
```

Original comment by `cbrug...@gmail.com` on 16 Jan 2010 at 11:58

**Comment by GoogleCodeExporter** [comment]
```
Damn, Peerblock for Mac. it's been long since there's no stable security 
software out there for the Mac. I'd LOVE a 
mac version of Peerblock. I'm running OSX 10.6.2. 

I know you've said you can't code the mac version because you lack people that 
program in Unix or Cocoa, but 
it's just a matter of asking around forums, posting options for volunteers and 
eventually some nice and smart 
mac developer will offer his or her help. (P.S.: Try for it to be Cocoa though, 
it's what makes Mac, well, Mac :P)
```

Original comment by `lusir...@gmail.com` on 18 Jan 2010 at 1:16

**Comment by GoogleCodeExporter** [comment]
```
I agree, I think there does need to be a MacOS version
```

Original comment by `rhodryk...@gmail.com` on 18 Jan 2010 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
Thanks for all the comments - we do have plans to start working on a Mac 
version of
PeerBlock, come the PeerBlock 2.0 timeframe.  As stated earlier (Comment #21), 
the
biggest problem we're confronted with is that the base code itself has a ways 
to go
before it's ready for a port to anything other than Windows.  Once we've
rearchitected and refactored our code, we will begin looking more into the Mac 
(and
Linux) side of things.

Don't worry, we'll get there eventually...  We realize that people are very
interested in PeerBlock for Mac, and would love to be able to satisfy your 
requests.
 We're just not quite there yet.
```

Original comment by `peerbloc...@gmail.com` on 18 Jan 2010 at 5:41

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Mac please im begging you fuck 1.2 and the other versions.... just shoot 
straight for the mac version
```

Original comment by `EsencyTr...@gmail.com` on 22 Jan 2010 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
Good to here that it's in the works! I will donate!
```

Original comment by `rpiango...@gmail.com` on 23 Jan 2010 at 6:48

**Comment by GoogleCodeExporter** [comment]
```
Peerguardian osx 1.62b no longer works.

List manager does not work. 
```

Original comment by `cooperst...@gmail.com` on 25 Jan 2010 at 12:36

**Comment by GoogleCodeExporter** [comment]
```
I want this to happen, but I'm sick to death of getting emails every time anyone
posts a "me too!" comment so I'm unticking the star for the issue. Consider my
comment a vote in favour. And yes, I do realise by commenting I am compounding 
the
issue and causing 165 unwanted emails to be sent. Sorry. If we all stop using 
the
star button, Google might fix the insanity of equating votes with interest in 
"me
too" emails.
```

Original comment by `m...@wwarby.com` on 25 Jan 2010 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
ditto
```

Original comment by `nikolaus...@gmail.com` on 25 Jan 2010 at 11:47

**Comment by GoogleCodeExporter** [comment]
```
I very much agree with comment 41. I'm all for porting PeerBlock to all the 
platforms
possible but I don't care for the e-mails from every single new comment.
```

Original comment by `Keefaet...@gmail.com` on 25 Jan 2010 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
:\ it *is* getting a bit tiresome everyday to receive an email about this.

But...

+Peerblock for Mac.
```

Original comment by `bryanats...@gmail.com` on 25 Jan 2010 at 12:08

**Comment by GoogleCodeExporter** [comment]
```
Ditto
```

Original comment by `cpt.shenanigans@gmail.com` on 25 Jan 2010 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
YES!! :)
```

Original comment by `futureso...@gmail.com` on 28 Jan 2010 at 3:38

**Comment by GoogleCodeExporter** [comment]
```
OSX Version please!
```

Original comment by `sheen....@gmail.com` on 30 Jan 2010 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
I will donate $25 once done!
```

Original comment by `tonyr...@gmail.com` on 31 Jan 2010 at 7:15

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
i would use and support your service via mac osx, please try to spend some 
resource
time in developing a build. thanks. 
```

Original comment by `morag.nu...@gmail.com` on 7 Feb 2010 at 12:39

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Ok. I'm sorry for adding to the pile of bs "please develop this" storm, but I'm 
tired
of the fricken emails. So. I unstarred this, but would like it and would 
appreciate
it if you idiots would NOT continue to put in your inane dribble about wanting 
it. As
it states here (http://www.peerblock.com/docs/faq#mac_port):

"If you're interested in a Mac port, you should "Star" that feature request.  
Not
only will it update you once we start to make progress on it, but when we're
prioritizing what to work on next we use the number of Stars an issue has as an
indication of how interested our user community is in something.  So consider 
it a
way for you to "vote" for that feature."

So just star the damn thing and stfu. Amusingly enough, I'm sure that that post 
was
how you got here in the first place! I'd be alright with these idiotic emails 
if it
was, "Yes, I'm a mac developer and want to work on it," cause at least that 
tells me
someone is moving in the right direction, but this, "I'd support you," and 
"Peerblock
please," crap is just damn annoying. 

Sorry for compounding the problem, but it really is dumb as hell.
```

Original comment by `lotek....@gmail.com` on 8 Feb 2010 at 7:41

**Comment by GoogleCodeExporter** [comment]
```
I would greatly appreciate a mac version!!!
```

Original comment by `jon.mcclay@gmail.com` on 10 Feb 2010 at 6:58

**Comment by GoogleCodeExporter** [comment]
```
Just  switched to Mac and a version for Snow Leopard would be very well 
appreciated !... :)
```

Original comment by `zetot...@gmail.com` on 13 Feb 2010 at 7:53

**Comment by GoogleCodeExporter** [comment]
```
I'd love a Mac version -- peerguardian 1.6b2 for Mac has so many bugs as to 
make it
unusable...  gave up on it after about 2 hours of using it.
```

Original comment by `jgrn307` on 19 Feb 2010 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
1. I'm a developer. (good)
2. I have a Mac Book to develop on. (good)
3. I've never worked on a project like this, but might be willing to help.
4. Unfortunately, most of my experience is on Sun/AIX with a little bit of 
Windows
thrown in. The only Mac development experience is a cheesy app (for me only) I 
wrote
for my iPhone.

...but like all the others, I'd love to see a port for OS X. So, what can I do 
to help?
```

Original comment by `mtsuro...@gmail.com` on 20 Feb 2010 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
Mac version!!! :)
```

Original comment by `GhostSta...@gmail.com` on 21 Feb 2010 at 1:46

**Comment by GoogleCodeExporter** [comment]
```
Awaiting a mac version asap!
```

Original comment by `property...@gmail.com` on 23 Feb 2010 at 4:14

**Comment by GoogleCodeExporter** [comment]
```
I too look forward to an OSX version of this application.
```

Original comment by `charliea...@gmail.com` on 25 Feb 2010 at 4:34

**Comment by GoogleCodeExporter** [comment]
```
would appreciate a mac version. 
```

Original comment by `cjmccorm...@gmail.com` on 27 Feb 2010 at 3:23

**Comment by GoogleCodeExporter** [comment]
```
I would like it if an OSX version were made available as well
```

Original comment by `excalibe...@gmail.com` on 27 Feb 2010 at 6:13

**Comment by GoogleCodeExporter** [comment]
```
YES.. we need a mac version!!! So many mac users these days anyways!
```

Original comment by `trausch...@gmail.com` on 7 Mar 2010 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
Please!!! I need a MAC version too!!!

```

Original comment by `carlo....@gmail.com` on 9 Mar 2010 at 12:17

**Comment by GoogleCodeExporter** [comment]
```
MAC Version Please
```

Original comment by `liebgold` on 15 Mar 2010 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
MAC Version Please
```

Original comment by `liebgold` on 15 Mar 2010 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Mac Version Please! and about the Buying a mac, cant you just build a couple of 
Hackintoshes???
```

Original comment by `Timothyj...@gmail.com` on 18 Mar 2010 at 1:41

**Comment by GoogleCodeExporter** [comment]
```
would be appreciated

```

Original comment by `jonbentl...@gmail.com` on 19 Mar 2010 at 11:11

**Comment by GoogleCodeExporter** [comment]
```
Hello,

I make use of Peerblock with Windows 7 (64 bits).

I thank you much for the support for the 64 bits.

I also do not make use badly of Mac OS X Snow Leopard, I would like to know if 
you
think of leaving one day a version PeerBlock for Mac OS X.

Cordially.

Bonjour,

J'utilise Peerblock avec Windows 7 (64 bits).

Je vous remercie beaucoup pour le support du 64 bits.

Je me sers aussi pas mal de Mac OS X Snow Leopard, je voudrais savoir si vous 
pensez
sortir un jour une version de PeerBlock pour Mac OS X.

Cordialement. 
```

Original comment by `tvi...@gmail.com` on 20 Mar 2010 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
I'm really interested to a mac edition of peer block. Peer Guardian is an 
abandoned
project and still in beta (for snow leopard) since a long time and I think it 
don't
work how it should.
```

Original comment by `Big.Jac...@gmail.com` on 20 Mar 2010 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
Mac version please!
```

Original comment by `kaiyoh...@gmail.com` on 22 Mar 2010 at 1:07

**Comment by GoogleCodeExporter** [comment]
```
Make one please!!!
```

Original comment by `Peter.O....@gmail.com` on 23 Mar 2010 at 12:21

**Comment by GoogleCodeExporter** [comment]
```
Come on guys this is what Mac people have been waiting for!!!!
```

Original comment by `Bashman59@gmail.com` on 25 Mar 2010 at 6:30

**Comment by GoogleCodeExporter** [comment]
```
I would love to see a mac version of this - 100%.
```

Original comment by `goat1...@gmail.com` on 26 Mar 2010 at 6:09

**Comment by GoogleCodeExporter** [comment]
```
In the meantime, while you wait for this, you can use this work around for Peer 
Guardian:

http://forum.transmissionbt.com/viewtopic.php?f=4&t=5468
```

Original comment by `ptpanz...@gmail.com` on 26 Mar 2010 at 6:27

**Comment by GoogleCodeExporter** [comment]
```
I'd love a version of PeerBlock for Mac OS X 10.6!
```

Original comment by `kevindmo...@gmail.com` on 28 Mar 2010 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
1. I'm a developer (Yay!)
2. I develop primarily for Unix and OS X (Yay!)
3. I have never(IIRC) done anything OS specific on any machine (boo!)
4. I am willing to help if I'm able :)
```

Original comment by `timo.j.sand@gmail.com` on 29 Mar 2010 at 4:40

**Comment by GoogleCodeExporter** [comment]
```
MAC VERSION PLEASE!!!!!!!
```

Original comment by `brettred...@gmail.com` on 29 Mar 2010 at 11:36

**Comment by GoogleCodeExporter** [comment]
```
I use Peer Guardian 1.6beta on my Mac without a problem (currently blocking 2.5
million IPs). after you install, i recommend removing ALL the presets/lists and
replace them with whichever bluetack ones float your boat.
http://forums.phoenixlabs.org/showthread.php?t=18902
```

Original comment by `Phi...@gmail.com` on 1 Apr 2010 at 7:20

**Comment by GoogleCodeExporter** [comment]
```
I use Windows/Mac, and Linux boxes. would be nice to have a universal app that 
worked
for both linux and mac.  Both are Unix based.  At any rate if any of you 
actually
want Peerblock for Mac you should use the DONATE button.
```

Original comment by `thomas.m...@gmail.com` on 2 Apr 2010 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
mac peerblock!
```

Original comment by `ekimia1987@gmail.com` on 2 Apr 2010 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
need mac peerblock.
```

Original comment by `aly...@gmail.com` on 6 Apr 2010 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
please port this, I have an older mac I could loan for testing purposes, not 
everyone 
uses 10.5 or 10.6 yet!!
```

Original comment by `dmitri...@gmail.com` on 9 Apr 2010 at 2:45

**Comment by GoogleCodeExporter** [comment]
```
would love to see a version for 10.6 osx
```

Original comment by `iamperip...@gmail.com` on 15 Apr 2010 at 9:41

**Comment by GoogleCodeExporter** [comment]
```
just giving this a bump! :)
```

Original comment by `david.w.matthews` on 19 Apr 2010 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
Mac OSX version please. :)
```

Original comment by `The.Rick...@gmail.com` on 19 Apr 2010 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
Thanks for all your work on this... BUMP!
```

Original comment by `tru2...@gmail.com` on 24 Apr 2010 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
i got a silver leopard for xmas so i need a mac peerblock too. i'll be watching 
this site 8) <--- smiley w/ 
glasses [i'm nearsighted]

```

Original comment by `nullifix@gmail.com` on 24 Apr 2010 at 5:45

**Comment by GoogleCodeExporter** [comment]
```
people, click on the star beside the Issue title to promote this idea!
```

Original comment by `salamaza@gmail.com` on 27 Apr 2010 at 8:51

**Comment by GoogleCodeExporter** [comment]
```
Os X please. Star clicked. Save us.
```

Original comment by `jim.be...@gmail.com` on 28 Apr 2010 at 10:19

**Comment by GoogleCodeExporter** [comment]
```
LOOK AT COMMENT 76! ^^^ and all the other ones.
we need this pretty bad devs.
```

Original comment by `bnorris1...@gmail.com` on 29 Apr 2010 at 8:29

**Comment by GoogleCodeExporter** [comment]
```
Yeah it would be great to have.. I'm sure many people would be happy to help 
code it..
I'm no programmer myself but I'm certainly sure there are others out there who 
would
help, and many of us would donate to help.
```

Original comment by `8Guadalupe8@gmail.com` on 2 May 2010 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
I'd help code it if I wasn't up to my neck in work and things, I've done some
programming, but I couldn't do it with my timeframe.. :-( But I'd really 
appreciate a
Ported mac version of PeerBlock.
```

Original comment by `nellavit...@gmail.com` on 2 May 2010 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
Yes please! Mac!!!!
```

Original comment by `jacqs...@gmail.com` on 14 May 2010 at 1:54

**Comment by GoogleCodeExporter** [comment]
```
would love to have Mac version.

Since You are allready using PG2 sorce code, maybe You could use PG 1.6b2 
sources of a Mac client?
After some coding it will work for sure, and no need to start from scratch.
```

Original comment by `vania.ni...@gmail.com` on 14 May 2010 at 2:27

**Comment by GoogleCodeExporter** [comment]
```
Mac version Please
```

Original comment by `fdnc...@gmail.com` on 15 May 2010 at 3:24

**Comment by GoogleCodeExporter** [comment]
```
A mac counterpart is definitely very much appreciated. Keep up the good work~~
```

Original comment by `zongii1...@gmail.com` on 15 May 2010 at 4:40

**Comment by GoogleCodeExporter** [comment]
```
Heard about Peerblock from my brother who uses PC. *Please* develop Peerblock 
for Mac
OSX (I'm currently running Mac OS 10.5.5).
```

Original comment by `ankita.awasthi@gmail.com` on 17 May 2010 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
Yes, please develop a Mac version! Can't you just turn one of your current 
desktops
into a Hackintosh for testing purposes?
```

Original comment by `paradigm...@gmail.com` on 21 May 2010 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
I use PeerGaurdian on my mac, but would LOVE a mac version of peerblock. I 
currently run 10.6.3, and tend to 
stay on top of every upgrade. I'd be more then happy to beta test when the time 
comes. 
```

Original comment by `ego...@gmail.com` on 23 May 2010 at 8:12

**Comment by GoogleCodeExporter** [comment]
```
mac version plox
```

Original comment by `nrrseri...@gmail.com` on 25 May 2010 at 11:53

**Comment by GoogleCodeExporter** [comment]
```
Peer block for the Mac OS (X)? .. yes please! I rather keep my pc clean and 
safe from those i do NOT want 
anywhere near it. 
```

Original comment by `FromOneT...@gmail.com` on 25 May 2010 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
We really do need a better working peerguardian2/peerblock. The version 
available for
mac is incredibly buggy and just barely works on osx 10.6. It's a shame too, 
since
the gui and how it works is great on windows. It would be even sexier on the 
mac. I
don't get why you guys don't just develop a cross platform package for this. 
Every OS
could  benefit from this relatively user friendly ip blocker. Windows does not 
have
monopoly on the OS market anymore either.

Cross platform, Open Source and free (donations encouraged), is the trend. Get 
with
it! =).
```

Original comment by `adamlo...@gmail.com` on 2 Jun 2010 at 6:59

**Comment by GoogleCodeExporter** [comment]
```
This is still a ways off, as per Comment #21.  We're slowly working towards it, 
but
have a LOT of work to do in the base code before we'll be ready to start 
working on a
Mac version.

Please remember that your comments will trigger emails to the over 300 people 
who've
Starred this issue.  You can Star it without adding a comment, and if you are
Starring it remember that you'll quickly get annoyed with all the "+1 me too" 
type
emails yourself!  So if you have something meaningful to add (like the fact that
you're a developer who'd be interested in working with us on an eventual Mac 
port, or
that you have an older Mac lying around that you'd be willing to donate to us) 
that's
great, definitely post a comment.  Otherwise, Starring the issue should be 
sufficient.

Thanks!  =;)

Believe me, we fully understand that a great many people are interested in a Mac
version.  And we'd love to be able to start working on it.  We're just not quite
there yet, still.
```

Original comment by `peerbloc...@gmail.com` on 4 Jun 2010 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
@peerblockproject:  Agreed!

:)  Sorry, 300+ people, I just had to.  You understand, right?
```

Original comment by `curtis07@gmail.com` on 4 Jun 2010 at 5:27

**Comment by GoogleCodeExporter** [comment]
```
The mac book pro (15' laptop) that I am using is yours once I finish my co-op 
and
purchase a new computer for myself. The monitor is broken clean off and thus I 
rely
on the DVI output connection. Will need a display or simply use VNC to work on 
it
headless. It's a real shame macs do not support wake on lan powered off =/. Dvd 
slot
is not reliable either for reading/burning cd/dvds.

It should be good enough for developing purposes however. Eventually I'll need a
mailing address to ship it to. For now I need it to complete a graphic
design/photography internship for the next 11 weeks or so, and you already 
mentioned
that it would not be awhile before the OS specific code is completely moved 
away from
the more OS independent code, so it works out conveniently. Keep up the good 
work!
```

Original comment by `adamlo...@gmail.com` on 4 Jun 2010 at 8:56

**Comment by GoogleCodeExporter** [comment]
```
This should definitely happen!
Cheers
```

Original comment by `sherloc...@gmail.com` on 3 Jul 2010 at 8:15

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Insulting us will not get a Mac version implemented any quicker.

The problem is that the code for our entire program is extremely 
Windows-specific right now.  Not only will the user-mode portion of the code 
need to be rewritten, but a new UI will need to be written, and a new driver 
will need to be written.

I'm deleting your comment.  Please keep discussions civil.
```

Original comment by `peerbloc...@gmail.com` on 12 Jul 2010 at 5:41

**Comment by GoogleCodeExporter** [comment]
```
Mac Please
```

Original comment by `DavidCHo...@gmail.com` on 15 Jul 2010 at 5:32

**Comment by GoogleCodeExporter** [comment]
```
Please mac
```

Original comment by `joseph.s...@gmail.com` on 23 Jul 2010 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
a mac version would be greatly appreciated as the current version of pg isnt 
kept up to date. mac users, take a look at http://www.hanynet.com/waterroof/ 
its a nice frontend to os x firewall, maybe this could be of help when making 
the mac version of peerblock?
```

Original comment by `manny.a....@gmail.com` on 26 Jul 2010 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
i think if i add one more entry to my hosts file my macbook will catch fire!! 
we need peerblock on mac.  thanks for all the hard work. (: 
```

Original comment by `rainnerl...@gmail.com` on 3 Aug 2010 at 5:02

**Comment by GoogleCodeExporter** [comment]
```
Although I am not the best person to spend time on this, I would love to learn 
about this tech and how to port it to a substrate that is compatible with OS X. 
I'd like to leave a comment to say that if a quality Mac port shipped which was 
as good or better than the Windows port, we tend to donate MUCH more. Thank you 
for making the internet a safer place & I'm tired of the bad stigmas these 
programs get from everyone. 
```

Original comment by `jasonbog...@gmail.com` on 6 Aug 2010 at 2:29

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Can't live with out a Mac version. Please make it soon. THANK YOU for your work.
```

Original comment by `solopert...@gmail.com` on 16 Aug 2010 at 1:00

**Comment by GoogleCodeExporter** [comment]
```
pretty pretty please for a mac version :)
```

Original comment by `jasonokeeffe` on 16 Aug 2010 at 9:32

**Comment by GoogleCodeExporter** [comment]
```
Would be great to have a Mac version, I'm still using PeerGuardian on the Mac 
but it is behaving kinda oddly recently.
```

Original comment by `Mid...@gmail.com` on 16 Aug 2010 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
Would be very interested in a Mac Version.
Cheers
```

Original comment by `OleyM...@gmail.com` on 18 Aug 2010 at 4:08

**Comment by GoogleCodeExporter** [comment]
```
Highly the Mac version
```

Original comment by `thomas.c...@free.fr` on 27 Aug 2010 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
another osx vote
```

Original comment by `Ricky1...@gmail.com` on 6 Sep 2010 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
here's one more vote
```

Original comment by `steve.wa...@gmail.com` on 6 Sep 2010 at 12:57

**Comment by GoogleCodeExporter** [comment]
```
You have my vote along with my offer to donate generously to get this worked on.
I currently maintain about 10 Windows 7 machines and 3 OSX machines.
```

Original comment by `Eigenbr...@gmail.com` on 6 Sep 2010 at 11:24

**Comment by GoogleCodeExporter** [comment]
```
Another Mac vote, thanks.
```

Original comment by `jeremycs...@gmail.com` on 22 Sep 2010 at 6:31

**Comment by GoogleCodeExporter** [comment]
```
So many offers for donations of money or computers, but not a single mention of 
using virtual machines. I'm not sure if that's because no one here knows about 
Oracle's free VirtualBox or if Mac OS in a VM is somehow insufficient for 
development and debugging, but it would certainly be a better use of our 
donations if you equip an already powerful PC or Linux box with Virtual Box and 
a Mac OS VM... It would also be available immediately, as opposed to having to 
wait for these machines to pour in.

I work in a heterogeneous IT environment and VirtualBox has more than proved 
its worth - it's free, you can't get better than that. For a measly $30, you 
can get a retail copy of Mac OS, which you technically can't install on 
anything that isn't Apple hardware (as stipulated by the EULA and enforced by 
the installer), but there are ways around it...
```

Original comment by `marc.eb...@gmail.com` on 24 Sep 2010 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Running a version of Mac OS in a VM is illegal, and as such isn't something we 
would want to undertake.  Also, one of the big things that needs to change when 
an OS is virtualized is the networking layer - you need special virtualized 
device-drivers to provide the network connection - so we wouldn't necessarily 
be testing against a "real" setup there.

Adam from comment #105 has graciously donated his old broken-screen MacBook, 
which we will be using as an official development platform.  We should 
hopefully soon be starting to work on rearchitecting PeerBlock to make it 
cross-platform capable.  It's still going to be a little while coming - lots of 
rework will be required - but between this and Linux/BSD VMs we should be able 
to at least get a certain baseline level of testing done.  

We will post status updates here, and will of course post here once we have 
something ready for beta test...
```

Original comment by `peerbloc...@gmail.com` on 25 Sep 2010 at 4:56

**Comment by GoogleCodeExporter** [comment]
```
Sounds awesome, thanks guys.
```

Original comment by `veloriu...@gmail.com` on 25 Sep 2010 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
thanks for thinking of the mac users. i'm currently using an old ibook g4 
(10.4) and hope that when a mac version becomes available that its compatible 
with 10.4.

thank you!
```

Original comment by `asembr...@gmail.com` on 27 Sep 2010 at 7:38

**Comment by GoogleCodeExporter** [comment]
```
MAC PEER BLOCK
```

Original comment by `Bdp...@gmail.com` on 7 Oct 2010 at 7:58

**Comment by GoogleCodeExporter** [comment]
```
Mac Peer Block please!
```

Original comment by `lukesik...@gmail.com` on 19 Oct 2010 at 8:57

**Comment by GoogleCodeExporter** [comment]
```
Mac Peer block would be great
```

Original comment by `thegoatw...@gmail.com` on 21 Oct 2010 at 11:03

**Comment by GoogleCodeExporter** [comment]
```
Thanks for considering porting a mac version.

I've only just moved over to a mac and still use PeerBlock on my windows 
machine. It would be awesome-o to have the luxury of PeerBlock on my mac too. =)

Thanks for a fantastic piece of software. I hope to see a mac version soon.
```

Original comment by `stewartg...@gmail.com` on 27 Oct 2010 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Mac Developer? Check
Willing to help? Check

Have a MacBook soon to upgrade to a MacBook Pro and am will in to help.  Let me 
know what I can do when the time comes.
```

Original comment by `dmar...@gmail.com` on 31 Oct 2010 at 5:51

**Comment by GoogleCodeExporter** [comment]
```
Peer Block for MAC would be awesome!
```

Original comment by `verda...@gmail.com` on 4 Nov 2010 at 7:38

**Comment by GoogleCodeExporter** [comment]
```
If you're interested in a Mac port, you should "Star" that feature request.  
Not only will it update you once we start to make progress on it, but when 
we're prioritizing what to work on next we use the number of Stars an issue has 
as an indication of how interested our user community is in something.  So 
consider it a way for you to "vote" for that feature.

Please do not add "+1 me too" style comments though!  

You can Star the issue without leaving a comment, and believe me we understand 
that people are looking for this feature.  
```

Original comment by `sean.con...@gmail.com` on 14 Nov 2010 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
Hell yeah!
```

Original comment by `philipp....@gmail.com` on 17 Nov 2010 at 10:02

**Comment by GoogleCodeExporter** [comment]
```
I want to vote for this without getting star emails, I'm sure you understand.

I strongly encourage doing mac development on a Hackintosh and letting the 
community test it on actual mac hardware. I need this badly because I use a pc 
running osx, I have to do my audio work on a Hackintosh due to windows firewire 
issues and apple being expensive issues.

ipcosx86 is really not hard to get installed and get driver "kexts" (kernel 
extensions) for, so any pc you already are using and a spare external hard 
drive to boot it from, and you can test stuff on osx.

don't be scared! not having a real mac is no excuse for not having a mac.
```

Original comment by `wozla...@gmail.com` on 21 Nov 2010 at 12:33

**Comment by GoogleCodeExporter** [comment]
```
As per comment #125, there are legal ramifications to setting up a "Hackintosh" 
environment.  And since we had to set up an official company in order to be 
allowed to purchase our code-signing certificate so we can generate a signed 
driver for use on x64 Windows Vista/7 installs, we need to run things as such.  
This means that breaking the law is kind of a no-no...

Again, however, one of our users donated his old Macbook machine (thanks again, 
Adam!), so we do now have at least one "real" Mac for development purposes.  We 
also have a couple Mac developers who've expressed interest in helping out with 
the Mac port.  And I myself - Mark, PeerBlock's lead developer - have recently 
been doing some Mac related development at work, so it won't be completely 
fumbling around in the dark.  

Work on this is currently blocked by our need to rearchitect our codebase in 
order to support cross-platform stuff in any way, shape, or form.  We'll get 
there, don't worry, it's just going to take a little time...
```

Original comment by `peerbloc...@gmail.com` on 21 Nov 2010 at 12:47

**Comment by GoogleCodeExporter** [comment]
```
Love Peer Block on windows, but would enjoy it for my macs!
MAC PORT!
```

Original comment by `trik...@gmail.com` on 23 Nov 2010 at 7:54

**Comment by GoogleCodeExporter** [comment]
```
i agree with comment 138
```

Original comment by `bartilot...@gmail.com` on 25 Nov 2010 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
have you guys looked at mono? http://www.mono-project.com/Main_Page this could 
help bring it to linux and mac
```

Original comment by `manny.a....@gmail.com` on 26 Nov 2010 at 7:54

**Comment by GoogleCodeExporter** [comment]
```
GNU/Linux Developer? Check
Willing to help? Check

New to Mac would love to see peerblock on mac
```

Original comment by `kartik...@gmail.com` on 28 Nov 2010 at 11:39

**Comment by GoogleCodeExporter** [comment]
```
Mac version pleeeeeease
```

Original comment by `yoshi...@gmail.com` on 7 Dec 2010 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
Mac Version would be really appreciated 
```

Original comment by `Schokoc...@googlemail.com` on 9 Dec 2010 at 10:00

**Comment by GoogleCodeExporter** [comment]
```
Please, let us a Mac version of this great app
```

Original comment by `sherezad...@hotmail.com` on 12 Dec 2010 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
A mac versin would be nice to have :)))) So please :))))
```

Original comment by `jasmin.m...@gmail.com` on 17 Dec 2010 at 12:01

**Comment by GoogleCodeExporter** [comment]
```
thank you for giving me another reason to convert back to windows.  but, a mac 
version would be helpful.
```

Original comment by `tepie.ba...@gmail.com` on 19 Dec 2010 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
yes by all means a  MAC version
```

Original comment by `chazbre...@gmail.com` on 23 Dec 2010 at 4:23

**Comment by GoogleCodeExporter** [comment]
```
I really, really, really, really, really want this for mac. Peer Guardian for 
mac is horrible. Just makes me feel safer. 
```

Original comment by `apla...@gmail.com` on 26 Dec 2010 at 10:41

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Really needing that mac version...thanks.
```

Original comment by `sbcal...@gmail.com` on 14 Jan 2011 at 11:26

**Comment by GoogleCodeExporter** [comment]
```
Definitely needed and it would definitely be appreciated.  Would also be really 
cool if we could get some kind of time table when you guys start seriously 
working on it so keep that in mind please! thanks and you guys are fighting the 
good fight!
```

Original comment by `Kendall....@gmail.com` on 16 Jan 2011 at 1:54

**Comment by GoogleCodeExporter** [comment]
```
For the love of god...PG2 rocked on PC, now I have a mac!  I want that same 
protection and peace of mind back, please release a mac version!!!
```

Original comment by `Visionar...@gmail.com` on 20 Jan 2011 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
Absolutely!
```

Original comment by `omni.sep...@gmail.com` on 27 Jan 2011 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
Just bought my first Mac and it's great but I miss you! Please come over to the 
dark side, it's lovely over here.
```

Original comment by `halleh...@yahoo.com` on 3 Feb 2011 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
same here, would love to have peerblock on the mac too !
```

Original comment by `kirat...@gmail.com` on 9 Feb 2011 at 1:52

**Comment by GoogleCodeExporter** [comment]
```
Ever since I switched to a Mac, the one thing I missed most if PeerBlock. 
Please release one soon! Thanks.
```

Original comment by `calvin...@gmail.com` on 17 Feb 2011 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
Linux-version? Yes, would push the donation button then (only on linux since a 
couple of months back)
```

Original comment by `Josef.An...@gmail.com` on 17 Feb 2011 at 7:42

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Mac version yes please!
```

Original comment by `Mr.Spick...@gmail.com` on 22 Feb 2011 at 8:28

**Comment by GoogleCodeExporter** [comment]
```
go mac version!

```

Original comment by `luca.fra...@gmail.com` on 24 Feb 2011 at 6:34

**Comment by GoogleCodeExporter** [comment]
```
when is this going to arrive? any dates set as a goal?  
```

Original comment by `apruler2...@gmail.com` on 1 Mar 2011 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
one more for a mac port!
```

Original comment by `tim.li...@gmail.com` on 3 Mar 2011 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
would love this for mac!
```

Original comment by `GCM...@gmail.com` on 5 Mar 2011 at 10:21

**Comment by GoogleCodeExporter** [comment]
```
One more for mac version. 
```

Original comment by `nyctothe...@gmail.com` on 6 Mar 2011 at 3:50

**Comment by GoogleCodeExporter** [comment]
```
Heck yes! A Mac version, please!
```

Original comment by `deweydec...@gmail.com` on 15 Mar 2011 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
Need a mac version as soon as possible.  There are numerous restrictions on the 
work network which must be surpassed.  As there is no viable competitor, I am 
left helpless until a mac supported has been released.  
```

Original comment by `aishwary...@gmail.com` on 16 Mar 2011 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
Mac version please, i'd donate for your commitment :)
```

Original comment by `thelastc...@gmail.com` on 16 Mar 2011 at 8:33

**Comment by GoogleCodeExporter** [comment]
```
if there are so many people wanting peerblock for mac, and the developers don't 
have a mac, why not send the mac-owners a test and have the developers collect 
the data?
or, if the developers really NEED a mac to actually work on the project, why 
not have a separate fund aside?  surly you can get something simple enough like 
a used mac or a mac mini to start off with.  
```

Original comment by `kamisets...@gmail.com` on 21 Mar 2011 at 6:13

**Comment by GoogleCodeExporter** [comment]
```
@168

Last thing I heard someone donated their old macbook for the purpose.
```

Original comment by `Robin.Vi...@gmail.com` on 5 Apr 2011 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
I am serious about this:
If a first working release of a Mac version is released, i'll send you a big 
fat $150 donation your way. 
```

Original comment by `markh...@gmail.com` on 8 Apr 2011 at 7:45

**Comment by GoogleCodeExporter** [comment]
```
I would love to see this
```

Original comment by `gal...@gmail.com` on 18 Apr 2011 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
I will send a donation as well!
```

Original comment by `keler...@gmail.com` on 19 Apr 2011 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
mac client please, I will donate as well.
```

Original comment by `philhe...@gmail.com` on 4 May 2011 at 9:09

**Comment by GoogleCodeExporter** [comment]
```
Need NEED need this!  Would love to be able to donate for such a worthwhile 
program.
```

Original comment by `PmRei...@gmail.com` on 13 May 2011 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
Will gladly donate for something like this. 
```

Original comment by `benselb...@gmail.com` on 14 May 2011 at 1:57

**Comment by GoogleCodeExporter** [comment]
```
Yeah I'd pay at least twenty bucks for one.
```

Original comment by `MattBenn...@gmail.com` on 20 May 2011 at 11:35

**Comment by GoogleCodeExporter** [comment]
```
Mac OSX client Please.
```

Original comment by `mruniver...@gmail.com` on 22 May 2011 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
Would love to see a Mac OSX version of this, peerguardian is not an alternative 
I want to use.
```

Original comment by `JDLea...@gmail.com` on 22 May 2011 at 6:54

**Comment by GoogleCodeExporter** [comment]
```
please get a mac version going

```

Original comment by `Pol...@gmail.com` on 26 May 2011 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
Please, we want mac os cliente of Peerblock.

Thanks
```

Original comment by `des.fran...@gmail.com` on 26 May 2011 at 10:55

**Comment by GoogleCodeExporter** [comment]
```
I tri-boot, this is a must.
```

Original comment by `tehsuse...@gmail.com` on 5 Jun 2011 at 4:21

**Comment by GoogleCodeExporter** [comment]
```
Mac OSX. Please.
```

Original comment by `nickpa...@cheapnet.it` on 11 Jun 2011 at 6:40

**Comment by GoogleCodeExporter** [comment]
```
Also looking forward to a Mac version. Don't ignore us! We have significant 
marketshare now!  :)
```

Original comment by `geoff.cr...@gmail.com` on 12 Jun 2011 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
Common guys, many of us would be more than willing to pay for an OSX version. 
Grab the latest Lion beta and start with that.
```

Original comment by `bruce.we...@gmail.com` on 20 Jun 2011 at 10:08

**Comment by GoogleCodeExporter** [comment]
```
PeerGuardian doesn't even work properly on OSX. Hopefully some people will 
become interested soon. I'll be posting this to reddit.com/r/apple to hopefully 
gain some support! We have over 50k readers and I know some of them will be 
interested.
```

Original comment by `chriswar...@gmail.com` on 1 Jul 2011 at 2:49

**Comment by GoogleCodeExporter** [comment]
```
PLZ add it.
```

Original comment by `trese...@gmail.com` on 1 Jul 2011 at 4:45

**Comment by GoogleCodeExporter** [comment]
```
mac mac mac...thank you!
```

Original comment by `capt...@gmail.com` on 1 Jul 2011 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
Would also appreciate an OSX version a lot.
```

Original comment by `LLBarne...@gmail.com` on 2 Jul 2011 at 7:09

**Comment by GoogleCodeExporter** [comment]
```
Me 2. Thanks. Also willing to donate if such version will be released.
```

Original comment by `YuriPetu...@gmail.com` on 3 Jul 2011 at 10:18

**Comment by GoogleCodeExporter** [comment]
```
Please Mac Version!!!
```

Original comment by `M.Ramsa...@googlemail.com` on 4 Jul 2011 at 12:40

**Comment by GoogleCodeExporter** [comment]
```
Peer Guardian has gone to the dogs! Please gives us a mac version! 
```

Original comment by `harry...@gmail.com` on 6 Jul 2011 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I would love to see Peer Block on Mac. Peer Guardian on Mac was a little buggy 
and slow.
```

Original comment by `MrTwinkl...@gmail.com` on 13 Jul 2011 at 9:25

**Comment by GoogleCodeExporter** [comment]
```
Would love to see a Mac OSX version of this, peerguardian is not an alternative 
I want to use.
```

Original comment by `bebebori...@gmail.com` on 17 Jul 2011 at 6:11

**Comment by GoogleCodeExporter** [comment]
```
PeerGuardian appears to be broken on OS X 10.7 (Lion). Please consider 
increasing priority of this feature request, to expedite PeerBlock becoming a 
viable replacement. Thank you.
```

Original comment by `joseph.e...@gmail.com` on 23 Jul 2011 at 3:43

**Comment by GoogleCodeExporter** [comment]
```
Peerblock for Lion please
```

Original comment by `kalj...@gmail.com` on 24 Jul 2011 at 7:28

**Comment by GoogleCodeExporter** [comment]
```
Confirming that as of OS X 10.7 Lion PeerGuardian is brokey brokey.
Is there any ETA update or progress on a Mac version of PeerBlock?
```

Original comment by `then...@gmail.com` on 25 Jul 2011 at 7:18

**Comment by GoogleCodeExporter** [comment]
```
PLEEEASE!
```

Original comment by `b...@geekoracle.com` on 27 Jul 2011 at 12:58

**Comment by GoogleCodeExporter** [comment]
```
ah, c'mon guys... over 8 months since the last post by a project member!!! if 
you're gonna do this, as Nike so elegantly stated, "just do it."
```

Original comment by `rljhai...@gmail.com` on 31 Jul 2011 at 8:44

**Comment by GoogleCodeExporter** [comment]
```
please very interested
```

Original comment by `dek...@gmail.com` on 2 Aug 2011 at 12:48

**Comment by GoogleCodeExporter** [comment]
```
I used to use PG on Leopard, but never could get it to function on Snow Leopard.
It would be nice to have a working blocker for Mac again.
```

Original comment by `OregonRe...@gmail.com` on 11 Aug 2011 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
Love to have this on the mac too !!!!
```

Original comment by `hfiore...@googlemail.com` on 12 Aug 2011 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
This would be fantastic thanks!
```

Original comment by `cameron....@gmail.com` on 13 Aug 2011 at 11:35

**Comment by GoogleCodeExporter** [comment]
```
Definitely would love to have this on the mac

```

Original comment by `Timmyd2...@gmail.com` on 29 Aug 2011 at 11:03

**Comment by GoogleCodeExporter** [comment]
```
Sorry, couldn't find the star icon....Would love to have a Mac version.
Thanks
```

Original comment by `michelle...@gmail.com` on 1 Sep 2011 at 12:57

**Comment by GoogleCodeExporter** [comment]
```
Michelle from comment 205: The star is at the top to the left of "Issue 161"
```

Original comment by `adam.whi...@gmail.com` on 7 Sep 2011 at 12:45

**Comment by GoogleCodeExporter** [comment]
```
Just a brief comment for the VM thought: VMs are supplanting their own 
networking into the equation, but that doesn't mean it's not "real". I feel the 
idea is actually more "standardized" I can still pick my specific card I want 
to simulate, and NAT vs. Bridged is handy... These happen with one click... 
Meanwhile too, I've heard tale that OSX Lion 10.7 is NOT illegal to run in a 
VM. This will help if your hardware can handle running even 2 VM copies of Lion 
(probably needs to handle Lion as a host first) 
http://images.apple.com/legal/sla/docs/macosx107.pdf See 2.B.iii . This will be 
a great opportunity for canned environments to test inside, without affecting 
the host. Cheers.
```

Original comment by `lookmomnohands` on 8 Sep 2011 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
Whats it gonna take to get this into development? A website to help support 
with donations to get it at least started? I'm down.
```

Original comment by `thelastc...@gmail.com` on 12 Sep 2011 at 10:11

**Comment by GoogleCodeExporter** [comment]
```
Please, for Mac!
```

Original comment by `mcinni...@gmail.com` on 15 Sep 2011 at 3:03

**Comment by GoogleCodeExporter** [comment]
```
Please, for Mac!
```

Original comment by `mcinni...@gmail.com` on 15 Sep 2011 at 3:04

**Comment by GoogleCodeExporter** [comment]
```
Hit me up for 20$ when you have a mac version, you have my email.
```

Original comment by `rahnjen...@gmail.com` on 25 Sep 2011 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
Being as this issue is two years old, maybe someone could start doing something 
about it?

Moreover, if you guys are tired of hearing the "me too" comments, here's a 
thought: consider perhaps porting PeerBlock to Mac OSX 10.7 and we'll stop 
bothering you. We (the people in this thread) have offered to give you the 
computers, developers, cash and infrastructure you need to make it happen, 
so... make it happen.

Thanks.
```

Original comment by `mcd...@gmail.com` on 8 Oct 2011 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
Seriously, when an issue's number of voters outpaces the number of votes for 
literally all of the other issues combined, that's when it's time to start 
thinking about implementing a fix for it.
```

Original comment by `mcd...@gmail.com` on 8 Oct 2011 at 5:02

**Comment by GoogleCodeExporter** [comment]
```
OS X... it is time. Thank you.
```

Original comment by `Jason.G....@gmail.com` on 17 Oct 2011 at 5:52

**Comment by GoogleCodeExporter** [comment]
```
Would be interested in both seeing and when possible aiding in bringing a 
PeerBlock for OS X to light. 

I'm a milddle-aged professional developer so have resources available both 
money-wise and with hardware and development skills.  Would be interested in 
discussing what your group would consider.  Not looking for any compensation - 
just to help the community.

```

Original comment by `aarondi1...@gmail.com` on 23 Oct 2011 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I am not a cpp developer. But I have macs running both Lion and Snow Leopard. 
Hence I would be able to compile and report back any errors.
```

Original comment by `jo...@joycebabu.com` on 3 Nov 2011 at 2:26

**Comment by GoogleCodeExporter** [comment]
```
Just want to say that this would be great. I use peer block on my Windows 
machines, and a Mac version would make me very happy.
```

Original comment by `jwood...@gmail.com` on 4 Nov 2011 at 5:10

**Comment by GoogleCodeExporter** [comment]
```
I don't know if this is the right place to post it, but I found a new version 
of peerguardian for mac: 1.7.0b. It seems to block stuff and to update lists 
works as well, has anyone else tried it? Still, I would love to see a MAC 
version of peerblock, used it on windows and loved it.
```

Original comment by `charlieb...@gmail.com` on 10 Nov 2011 at 8:36

Attachments:
- [guardian.png](https://storage.googleapis.com/google-code-attachments/peerblock/issue-161/comment-219/guardian.png)

**Comment by GoogleCodeExporter** [comment]
```
Please create a mac port of this software. However please it must be a 
universal binary. Because there are lots of people who use the G5 series of 
macs.
```

Original comment by `cole.hen...@gmail.com` on 30 Nov 2011 at 8:32

**Comment by GoogleCodeExporter** [comment]
```
one more vote please!
```

Original comment by `kingfe...@gmail.com` on 10 Dec 2011 at 5:30

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
 Please give us a port, there doesn't seem to be a blocklist manager for OS X anywhere. I'd be glad to help any way I can!!
```

Original comment by `will.mci...@gmail.com` on 14 Dec 2011 at 5:19

**Comment by GoogleCodeExporter** [comment]
```
let's make it happen, it's been two years! thanks!
```

Original comment by `showmeyo...@gmail.com` on 22 Dec 2011 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
Yeah I'm still waiting for an update on this issue.
```

Original comment by `HJk...@gmail.com` on 25 Dec 2011 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
ditto, running lion, just discovered Peerguardian using Version 1.6b2 (721) and 
the blocklists from http://www.iblocklist.com/lists.php
but I understand peerguardian is no longer being developed and it looks like 
Bluetack blocklist http://bluetack.co.uk/ needs support as well.

```

Original comment by `mscott...@gmail.com` on 10 Jan 2012 at 6:46

**Comment by GoogleCodeExporter** [comment]
```
I too would greatly appreciate a mac version of Peerblock. Pretty pleez :)
```

Original comment by `1largemo...@gmail.com` on 23 Jan 2012 at 11:44

**Comment by GoogleCodeExporter** [comment]
```
I would love a mac version, willing to donate! i know alot of others would be 
for this as well!
```

Original comment by `TJM.midd...@gmail.com` on 25 Jan 2012 at 5:38

**Comment by GoogleCodeExporter** [comment]
```
So, question:  Will PeerBlock work with Windows emulator programs like Fusion?  

I've been researching whether I want to boot camp my MAc or go with Fusion of 
Parallels... Having PeerBlock work in Parallels/Fusion would pretty much make 
my mind up for me.

Thanks!
```

Original comment by `rkrau...@gmail.com` on 26 Jan 2012 at 3:39

**Comment by GoogleCodeExporter** [comment]
```
WTF is the holdup? Make a MAC version NOW!
```

Original comment by `phix...@gmail.com` on 30 Jan 2012 at 1:00

**Comment by GoogleCodeExporter** [comment]
```
Mac version please!
```

Original comment by `MSLaf...@gmail.com` on 8 Feb 2012 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
mac please!
```

Original comment by `acplu...@gmail.com` on 13 Feb 2012 at 12:15

**Comment by GoogleCodeExporter** [comment]
```
please can we have a mac version
```

Original comment by `davidmor...@googlemail.com` on 13 Feb 2012 at 11:05

**Comment by GoogleCodeExporter** [comment]
```
As OSX gains more and more market share, it makes sense to extend Peerblock to 
them, for everybody's safety.
```

Original comment by `fruiz...@gmail.com` on 18 Feb 2012 at 5:04

**Comment by GoogleCodeExporter** [comment]
```
Max version pretty please
```

Original comment by `mattcro...@gmail.com` on 19 Feb 2012 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
Hoping for Mac version..... 
```

Original comment by `jonv...@gmail.com` on 27 Feb 2012 at 8:34

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Its been 3 years since you knew there was demand for a Mac version.
Just charge $4.99 for it. Old windows users would pay it by the thousands and 
you'd make your money back for your efforts. In the 3 years. 3 iPhones have 
been released, 3 iPads. A new version of the Mac OS and Windows have been 
released and even a whole new version of Adobe CS.
They can do it. You can write a program for Mac. It's not so hard it takes over 
3 years to make. You've clearly not been working on it. Come on guys. Just sell 
it. And make it. Keep the windows one free. That way when they migrate over to 
a Mac they will make the purchase. It would be a no-brainer. Stop stalling and 
please do it. 

The Best iPhone Tutorials on the Web
A5six.com
YouTube.com/a562178
```

Original comment by `a562...@gmail.com` on 5 Mar 2012 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
I completely agree with whoever made comment 238, you summed it up perfectly! I 
clicked on the link in the Peerblock FAQ expecting to find recent comments only 
to realize the comments go back to 2009 and it's now March 2012. So.... how 
hard are you working on bringing out a Mac version? You mentioned your 
"developer" was about to purchase a Mac so she/he could write the code....Price 
on Apple computers has come waaaay down in the last three years -- I've seen 
some Mac laptops advertised under $700 CDN. Anyway, good luck with it, and get 
on it! If you can't afford a Mad for your developer, hell, even I can throw a 
few dollars to you and I'm poor (which is why I download torrents in the first 
place!) TY :)

```

Original comment by `HeidiSta...@gmail.com` on 5 Mar 2012 at 3:38

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I have been a financial supporter since the old Methlabs days in 2002 (not a 
lot of cash but about $100 total). I would love to see a MAC version now as I 
have added a mac to my arsenal (due to video editing). I no longer need 
antivirus but get very nervous as to who is trying to access my computer. I use 
Peerblock all the time on my PC, even though I am not doing p2p. It is scary to 
view what is trying to hit my system. So please let us know when a MAC version 
comes out. I will gladly send more cash. (around $50 to help out as I 
appreciate it being free) Thanks Again, E.
```

Original comment by `faceless...@yahoo.com` on 18 Mar 2012 at 10:15

**Comment by GoogleCodeExporter** [comment]
```
I agree, I own a PC and mac, I would love a Mac version.
```

Original comment by `koltchan...@gmail.com` on 25 Mar 2012 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
Guys you should stop posting here. They haven't worked on the PC version for 
1,5 years. So obviously they have ditched the project and thus we won't see a 
Mac version, ever.
```

Original comment by `Nick.Bo...@gmail.com` on 7 Apr 2012 at 2:35

**Comment by GoogleCodeExporter** [comment]
```
There certainly looks to be a lot of support for this idea - I would like to 
add my voice the the heap.
```

Original comment by `Sfac...@gmail.com` on 7 Apr 2012 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
Yes, we are still working on things - it's just going pretty slowly right now, 
Real Life taking precedence and such.  We have a bunch more stuff on the 
roadmap, Mac support included.

We're still working to re-architect things in such a way that it can be used 
cross-platform.  In fact we need to do this even to better support Windows, and 
once that's done we'll be able to start figuring out how to best provide Mac 
support.  And linux, ideally even DD-WRT/OpenWRT based routers.  Hopefully even 
Android devices.  

So no, we haven't "ditched the project".  You just need to remember - we have 
jobs, children, families, etc.  While the donations we receive are gratefully 
accepted, we don't receive nearly enough to support any of us for any 
meaningful amount of time.  It's primarily a "labor of love".

Again, it'll be coming eventually . . . you just need to bear with us.  It's 
frustrating, I know, but there's little we can do to move things along more 
quickly right now.  Sorry.  We understand there's a lot of interest in a Mac 
version of PeerBlock, and fully intend to try and fulfill those wishes as best 
we can.
```

Original comment by `peerbloc...@gmail.com` on 8 Apr 2012 at 4:44

**Comment by GoogleCodeExporter** [comment]
```
I'm not too familiar with it so I just based my post on the latest stable 
release (for Windows).

Thanks for the update :)
```

Original comment by `Nick.Bo...@gmail.com` on 8 Apr 2012 at 5:04

**Comment by GoogleCodeExporter** [comment]
```
If you charged $5 on the Mac App Store, I would pay it. I'm sure many would.
```

Original comment by `cubex...@gmail.com` on 8 Apr 2012 at 5:58

**Comment by GoogleCodeExporter** [comment]
```
Totally. Put it in the Mac App store and get some green for your labor. I know, 
that doesn't create time in your schedules ex nihilo, but it would be a way to 
get back some capital for all that you  have invested in the project. I'd 
certainly drop $5.00 for it.

And kudos for letting us know what's going on. RL is priority uno, and we get 
that. If there's anything we can do to help, ask us. I'm certain there's a 
number of people in the user/follower community who can assist in a variety of 
ways.

Jake
```

Original comment by `jwood...@gmail.com` on 8 Apr 2012 at 6:28

**Comment by GoogleCodeExporter** [comment]
```
I'm not sure what peer block is. Did I email you a long time ago?

On Sunday, April 8, 2012, wrote:
```

Original comment by `danielvi...@gmail.com` on 8 Apr 2012 at 9:01

**Comment by GoogleCodeExporter** [comment]
```
To peerblockproject at gmail:

I don't understand how this is so fundamentally complicated. OSX ultimately 
runs on a UNIX source base. How is this much more difficult than skinning 
pf/pfctl for OSX to look like PeerBlock in Cocoa?

Stop trying to reinvent the wheel and just give the people what they want, ffs. 
Even if it doesn't work exactly like the Windows version, you have people who 
are willing to PAY you to develop this product. Apple users seem to have an 
endless supply of money to spend on their apps and they love paying off 
developers of their favorite products.

Mac firewalls are sorely needed and largely garbage, and pretty much the only 
option right now is Little Snitch. Fill the need, make it look like whatever, 
even if it's unique to the Mac, and practically overnight you'll have 10,000 
users paying you $5 a pop to buy the "better"/"more intuitive"/"touch-friendly" 
version of PeerBlock for OSX, thereby financing your project and redevelopment 
needs.
```

Original comment by `mcd...@gmail.com` on 9 Apr 2012 at 5:11

**Comment by GoogleCodeExporter** [comment]
```
Honestly, if you think it's all that hard, watch as I steal the project and 
thunder right out from underneath you and take that $50K+ for myself. PeerBlock 
has all of, oh, I dunno, twenty options to accomodate for in the GUI? The rest 
is handled by the engine, which as I just mentioned can be done entirely via PF 
rules (as PF has an outstanding featureset and if it ain't broke don't fix it).

Not trying to be rude; just stating the obvious. If you don't accomodate the 
needs of the market, be prepared for BeerPloq to show up in the App Store at 
the hands of some opportunistic developer like myself.
```

Original comment by `mcd...@gmail.com` on 9 Apr 2012 at 5:16

**Comment by GoogleCodeExporter** [comment]
```
Please bring out a Mac version ASAP - freeware or paid version, it's in demand. 
Thx

```

Original comment by `Andrews....@gmail.com` on 18 Apr 2012 at 12:10

**Comment by GoogleCodeExporter** [comment]
```
I would love to see this running on OS X 10.7 (hopefully 10.8) Many users we 
have tend to stay up to date on software. 
```

Original comment by `Perkele...@gmail.com` on 27 Apr 2012 at 9:14

**Comment by GoogleCodeExporter** [comment]
```
Hey, thanks for the informative reply.
Just so you understand (if my tone didn't reflect this, let's clear that up
RIGHT now! :)), I'm in awe of PeerBlock and that was a quick request for
you guys to consider adding OSX support. It wasn't a screed to abandon your
children (and/or perfectly good bottles of vodka/scotch/Absinthe staring at
you from the shelf wondering why you've abandoned them).
Just a gentle post saying, Hey, there's support for your making it
available for Mac, when you get the time.

Again, thanks so much!

It looks like the media cartels and our gov't are cheerfully marching in
unison to lock down the internet here in the US, and projects like
Peerblock go a far way to even the playing field. Thanks so much for
everything!

Jefe

On Sun, Apr 8, 2012 at 9:45 AM, <peerblock@googlecode.com> wrote:
```

Original comment by `jefe...@gmail.com` on 1 May 2012 at 4:02

**Comment by GoogleCodeExporter** [comment]
```
Keep up the awesome work and I hope to see peerblock on osx soon
```

Original comment by `prana...@gmail.com` on 3 May 2012 at 9:19

**Comment by GoogleCodeExporter** [comment]
```
@mcdade: You keep complaining and saying how easy it it and I don't think I 
seen you contribute anything at all apart from spamming the 1000+ users who 
starred this issue.

@everyone: Adding a "PeerBlock for Mac please!" won't help this issue go any 
faster.

None of use have access to a Mac let alone OS X 10.7.

There is already PeerGuardian OS X which should work although maybe not on OS X 
10.7.
```

Original comment by `nightstalkerz` on 10 May 2012 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
i need that!
```

Original comment by `pyramidc...@gmail.com` on 30 May 2012 at 6:35

**Comment by GoogleCodeExporter** [comment]
```
Yes Please !!
```

Original comment by `he...@quekett.me` on 21 Jun 2012 at 9:41

**Comment by GoogleCodeExporter** [comment]
```
Ditto for me, Lion and then Mountain Lion when available
```

Original comment by `5h4rp...@gmail.com` on 27 Jun 2012 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
I would also love to have this available !
```

Original comment by `sk8board...@gmail.com` on 11 Jul 2012 at 1:40

**Comment by GoogleCodeExporter** [comment]
```
YES please
```

Original comment by `bur...@gmail.com` on 16 Jul 2012 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
I have a mac and Xcode but not sure about how to get involved, if I could help 
I'd like to
```

Original comment by `oliver.k...@gmail.com` on 22 Jul 2012 at 12:44

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Mountain Lion is to be released tomorrow. Hope to see a PeerBlock for macs 
soon! <3
```

Original comment by `rx71...@gmail.com` on 25 Jul 2012 at 5:30

**Comment by GoogleCodeExporter** [comment]
```
Wouold love to be able to use Peerblock for a mac laptop! 
```

Original comment by `muellerp...@gmail.com` on 29 Jul 2012 at 4:41

**Comment by GoogleCodeExporter** [comment]
```
Would be great to have a mac version!
```

Original comment by `stir...@gmail.com` on 14 Sep 2012 at 2:35

**Comment by GoogleCodeExporter** [comment]
```
Mac version would be wonderful

by tomd.horton@gmail.com
```

Original comment by `TomD.Hor...@gmail.com` on 30 Sep 2012 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
Great product, please bring it to the Mac
```

Original comment by `dawalle...@gmail.com` on 14 Oct 2012 at 10:16
